Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1498,8 +1498,8 @@
         "com.apple.webinspectord.availability_check"))
 
 (with-filter (state-flag "WebContentProcessLaunched")
-    (allow mach-task-special-port-get (with report) (with telemetry))
-    (allow mach-task-special-port-set (with report) (with telemetry)))
+    (deny mach-task-special-port-get (with telemetry))
+    (deny mach-task-special-port-set (with telemetry)))
```

Comment:
Similar question from a prior review: If we care enough to get telemetry, do we not want a local report? Telemetry may not get captured if lots of other events are being triggered, so we might miss some things.

Reply:
That is a good point. We will get local reports by default from a violation, I believe, so the report modifier may not be needed.

Thanks for reviewing!

Reply:
Oh, of course. 'allow' needs an explicit report command to get the report. Denials always get a report. Thank you for reminding me! :-)